### PR TITLE
created new grid layout and changed ReviewList

### PR DESCRIPTION
**Pull request by andreykishtov**
create better layout on location page and made new file with styled components.
changed reviewlist component by making Wrapper inside wrapper.

### CODE REVIEW

**Comment by Alaev** [comment]
@andreykishtov review first https://github.com/Alaev/muncher-back/pull/9

File: src/components/Reviews/ReviewList.js [hunk 1]
```diff
@@ -6,22 +6,34 @@ import 'antd/lib/timeline/style/css';
...
+  ));
   return (
-    <div>
+    <Wrapper>
       <h1>Reviews</h1>
-      <Timeline pending={<a href="#">See more</a>}>
-        {Reviews}
-      </Timeline>
-    </div>
+      <Timeline pending={<a href="#">See more</a>}>{Reviews}</Timeline>
```
Comment (Alaev) [inline src/components/Reviews/ReviewList.js]: why do you have link to the same page?

**Comment by Alaev** [comment]
@andreykishtov 
Look at the images, the new design just looks not good.
images are to big review list to large. why did you make the change to the product?
Also when you go to the `/location` it throws an error.

we should to a product session before doing this kind of changes.

**Comment by Alaev** [comment]
# Before
screen shot 2017-12-24 at 23 03 21

# After
screen shot 2017-12-24 at 22 59 54
screen shot 2017-12-24 at 22 59 37

File: src/components/Reviews/ReviewList.js (hunk 1 above)
Comment (andreykishtov) [reply src/components/Reviews/ReviewList.js]: i dont know its not my component

**Comment by andreykishtov** [comment]
location path was fixed in other pull request

**Pull request closed without merging**